docs/best-practices/fine-grained-cpu-orchestration.md

```diff
@@ -0,0 +1,259 @@
+# Coordinated sharing of CPU resources in Colocation Scenarios - Fine-grained CPU Orchestration
+
+## Introduction
+
+In a cloud-native environment, cluster providers often deploy different types of workloads in the same cluster, leveraging different peak effects of different services to achieve time-sharing multiplexing of resources and avoid resource waste. However, colocation of different types of workloads often leads to resource competition and mutual interference. The most typical scenario is the colocation  of online and offline workloads. When more computing resources are occupied by offline workloads, the response time of online loads will be affected; when more computing resources are occupied by online workloads for a long time, the task completion time of offline workloads cannot be guaranteed. This phenomenon belongs to the Noisy Neighbor problem. 
```

Comment:
Recommend: `There are many different types of workload in Kubernetes cluster, and.........`